Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)
+        {
+            System.out.println(arr[i]);
+        }

```

Comment:
**Suggestion**: Consider adding error handling or a check to ensure the array is not null or empty before attempting to print even numbers to avoid potential NullPointerExceptions or unexpected behavior. <br> **Code Suggestion**: 
 ```
 +        if (arr == null || arr.length == 0) return;
 +        for (int i = 1; i < arr.length; i += 2)
 ```